integration-tests/plugins/package.json

```diff
@@ -15,14 +15,15 @@
     "@medusajs/currency": "workspace:^",
     "@medusajs/customer": "workspace:^",
     "@medusajs/event-bus-local": "workspace:*",
-    "@medusajs/inventory": "workspace:^",
+    "@medusajs/inventory": "latest",
```

Comment:
since we have removed the plugin shouldn't we remove the test and deps here? their will still exists in the v1 branch